internal/cloud/atlas/client.go

```diff
@@ -23,6 +23,7 @@ const (
 // Client is a MongoDB Cloud Atlas client
 type Client interface {
 	Groups() ([]Group, error)
+	FetchGroups(path string, options api.RequestOptions) ([]Group, error)
```

Comment:
[question] I'll relent that this needs to be its own function and exported for the time being, but does it also need to be part of the interface?  i'd like to limit the impact of "coding for testing purposes" here

Reply:
I believe it does need to be part of the interface ... in order to be able to access *client

Reply:
yeah you're right... back to the drawing board on this one.  I'd hold off on the merge for now, but I really want to get this into a less hacky-state